pepper-apis/dss-database/src/main/resources/db-changes/seed/DDP-7291-add-error-present-activity-instance-status.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
added new status translations. en, es, ru for now as per basil support.
need to add more depending on languages supported by fon